app/data_models/answer.py

```diff
@@ -69,13 +74,13 @@ def escape_answer_value(value: str) -> Markup: ...  # pragma: no cover
 
 @overload
 def escape_answer_value(
-    value: Union[None, int, Decimal]
-) -> Union[None, int, Decimal]: ...  # pragma: no cover
+    value: None | int | Decimal,
```

Comment:
Comma not needed?

Reply:
This gets flagged on the lint check and added when reformatted by black 🤔

Reply:
I guess it's not such a big problem, doesn't change the syntax as it's only a type hint.